backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts

```diff
@@ -1,5 +1,6 @@
 import { User as DbUser } from '@entity/User'
 import { Event as DbEvent } from '@entity/Event'
+/* eslint-disable-next-line import/no-cycle */
```

Comment:
disable this rule?

Reply:
Solution I implemented now: disable rule

Reply:
Better solution proposed in https://github.com/gradido/gradido/pull/2905